_posts/2022-06-06-homebrew-3.5.0.md

```diff
@@ -0,0 +1,168 @@
+---
+title: 3.5.0
+author: MikeMcQuaid
+redirect_from: /blog/3.5.0/
+---
+Today I'd like to announce Homebrew 3.5.0. The most significant changes since 3.4.0 are .
```

Comment:
`The most significant changes since 3.4.0 are .`
Not yet determined or missing unintentionally?

Reply:
Both 😁. PR's still a draft.